src/index.ts

```diff
@@ -0,0 +1,349 @@
+        defaultOption: true,
+        defaultValue: [""]
+    },
+    { name: "help", alias: "h", type: Boolean, defaultValue: false },
+    { name: "quiet", alias: "q", type: Boolean, defaultValue: false },
+    { name: "no-box", alias: "Q", type: Boolean, defaultValue: false },
+    { name: "add-tld", alias: "t", type: String, defaultValue: [""], multiple: true },
+    { name: "dry-run", alias: "D", type: Boolean, defaultValue: false }
+]) as arguments;
+
+if (arguments_.version) {
+    console.log("v3.1.1");
+    console.log(arguments_);
+    process.exit(0);
+}
+
+if (arguments_.help) {
+    // Show usages if has argument "--help"
+    console.log(usage);
+    process.exit(0);
+}
+
+if (!arguments_["dry-run"] && arguments_.quiet && arguments_.verbose) {
+    // quiet and verbose cannot be used at the same time without --dry-run
+    console.warn(`${chalk.yellowBright.inverse.bold(`  ${figures.warning}  `)} ${chalk.bold("--quiet")} and ${chalk.bold("--verbose")} cannot be used at same time! Replaced with default value!`);
+    [ arguments_.quiet, arguments_.verbose ] = [ false, false ];
+}
+
+const addTld = [ "co.jp", "or,jp", "ne.jp", "ac.jp", "ad.jp", "ed.jp", "go.jp", "gr.jp", "lg.jp" ];
+
+const requestGet = () => {
+    let lockFlag = false;
+
+    const spinner = arguments_.verbose ? ora("Fetching top-level domains information from IANA...") : undefined;
+
+    if (spinner) {
+        spinner.start();
+    }
+
+    https.get("https://data.iana.org/TLD/tlds-alpha-by-domain.txt", (response) => {
+        response.on("data", (chunk) => {
+            if (!lockFlag) {
+                if (spinner) {
+                    spinner.succeed(`Fetching top-level domains information from IANA...${chalk.greenBright("Success")}`);
+                }
+
+                main([
+                    ...(`${chunk}`
+                        .toLowerCase()
+                        .split(/\r\n|\n|\r/)
+                        .filter(tld => !tld.startsWith("#") || !tld)),
+                    ...addTld
+                ]);
+            } else {
+                lockFlag = true;
+            }
+        });
+    }).on("error", (error) => {
+        (async () => {
+            if (spinner) {
+                spinner.fail(`Fetching top-level domains information from IANA...${chalk.redBright("Failed")}`);
+            }
+
+            console.error(error);
+            console.log(`${chalk.redBright(figures.pointer)} ${chalk.bold("Please report this issue to")} ${terminalLink("Github Issues", "https://github.com/P2P-Develop/TopDomainChecker/issues")}`);
+
+            let copyAnswer = { cp: false };
+
+            copyAnswer = await Enquirer.prompt({
+                type: "confirm",
+                name: "cp",
+                message: "Copy stack-trace to clip board?"
+            });
+
+            if (copyAnswer.cp) {
+                ncp.copy(error);
+            }
+        })();
+    });
+};
+
+// Ping -> OK domains
+const aliveDomain = [""];
+
```

Comment:
Codacy found an issue: [There should be no space after '['.](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6364165)